packages/server/database/migrations/20200824105351-pokerTemplate.ts

```diff
@@ -0,0 +1,200 @@
+import {R} from 'rethinkdb-ts'
+import shortid from 'shortid'
+
+const createdAt = new Date('2020-08-24')
+const templates = [
+  {
+    createdAt: createdAt,
+    id: 'estimatedEffortTemplate',
+    isActive: true,
+    name: 'Estimated Effort',
+    teamId: 'aGhostTeam',
+    updatedAt: createdAt,
+    scope: 'PUBLIC',
+    orgId: 'aGhostOrg'
+  },
+  {
+    createdAt: createdAt,
+    id: 'wsjfTemplate',
+    isActive: true,
+    name: 'Weighted Shortest Job First',
+    teamId: 'aGhostTeam',
+    updatedAt: createdAt,
+    scope: 'PUBLIC',
+    orgId: 'aGhostOrg'
+  }
+]
+
+const dimensions = [
+  {
+    id: 'eeStoryPointsDimension',
+    name: 'Story Points',
+    teamId: 'aGhostTeam',
+    templateId: 'estimatedEffortTemplate',
+    scaleId: 'fibonacciScale',
+    createdAt,
+    updatedAt: createdAt
+  },
+  {
+    id: 'wsjfStoryPointsDimension',
+    name: 'Story Points',
+    teamId: 'aGhostTeam',
+    templateId: 'wsjfTemplate',
+    scaleId: 'fibonacciScale',
+    createdAt,
+    updatedAt: createdAt
+  },
+  {
+    id: 'wsjfStoryValueDimension',
+    name: 'Story Value',
+    teamId: 'aGhostTeam',
+    templateId: 'wsjfTemplate',
+    scaleId: 'fibonacciScale',
+    createdAt,
+    updatedAt: createdAt
+  }
+]
+
+const scales = [
+  {
+    id: 'fibonacciScale',
+    createdAt,
+    updatedAt: createdAt,
+    name: 'Fibonacci',
+    values: ['1', '2', '3', '5', '8', '13', '21', '34'],
+    teamId: 'aGhostTeam'
+  },
+  {
+    id: 'tshirtSizeScale',
+    createdAt,
+    updatedAt: createdAt,
+    name: 'T-Shirt Sizes',
+    values: ['XS', 'SM', 'M', 'L', 'XL'],
+    teamId: 'aGhostTeam'
+  },
+  {
+    id: 'fiveFingersScale',
+    createdAt,
+    updatedAt: createdAt,
+    name: 'Five Fingers',
+    values: ['1', '2', '3', '4', '5'],
```

Comment:
@mattkrick do we include `?` and `Pass` as dumb values so that they can remove or edit the color? Do we add a color palette value yet? Or is this really just stubbing it out?

Reply:
mmmmm, both valid points.
i think we do need the ? and Pass. that way folks can remove them if they want.
we do need colors attached to those values.
we may need a "value" attached to the "label". so if e.g. someone renames "Pass" to their native language we still treat it as the pass card & it gets sorted to the bottom of the list.
maybe values is `{label, value, color}`
i'm tempted to keep it denormalized & just have this be an array of objects to avoid a whole new table just for values.

Reply:
That sounds good, maybe this gets reworked in #4057